spec/models/rental_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Rental, type: :model do
```

Comment:
Since RSpec is configured with `infer_file_type_from_spec_location!` ([source](https://github.com/josephkelley/probable-engine/commit/e58b45063c538816c0378bedc996ec914c89d38a#diff-e11bdaf9116be3911e9557b69e242370R15)), you can actually just write

```ruby
describe Rental do
```

here.